packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1685,4 +1685,12 @@ class _DropdownButtonFormFieldState<T> extends FormFieldState<T> {
       setValue(widget.initialValue);
     }
   }
+
+  @override
+  void reset() {
+    final DropdownButtonFormField<T> dropdownButtonFormField = widget as DropdownButtonFormField<T>;
+    assert(dropdownButtonFormField.onChanged != null);
+    dropdownButtonFormField.onChanged!(widget.initialValue);
+    super.reset();
```

Comment:
In CupertinoTextFormFieldRow, `reset` calls `super` on the first line.  Should this one also call it on the first line?